docs/source/examples.daisy_chain_gap.ipynb

```diff
@@ -0,0 +1,746 @@
+    "\n",
+    "The main steps are: \n",
+    "\n",
+    "1. Create some molecules\n",
+    "2. Run GFN2-xTB MD\n",
+    "3. Filter by force components\n",
+    "4. Calculate global SOAP descriptor\n",
+    "5. Perform CUR decomposition to select diverse-ish training and testing sets\n",
+    "6. Fit a GAP potential\n",
+    "7. Evaluate structures with GAP\n",
+    "8. Plot atomization energy and force component correlation plots. \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports \n",
+    "\n",
+    "In addition to standard packages or wfl dependencies, we make use of three external packages: \n",
+    "\n",
+    "- quip and quippy which provide interface for fitting and evaluating GAP. \n",
+    "  \n",
+    "  Documentation: https://pypi.org/project/quippy-ase/ \n",
+    "  \n",
+    "  Installation: `pip install quippy-ase`\n",
+    "\n",
+    "\n",
+    "- GFN2-xTB: a semi-empirical method designed for molecular systems, used as a reference method. \n",
+    "\n",
+    "  Documentation: \n",
+    "  - https://xtb-docs.readthedocs.io/en/latest/contents.html\n",
+    "  - https://xtb-python.readthedocs.io/en/latest/\n",
+    "\n",
+    "  Installation: `conda install -c conda-forge xtb-python`\n",
+    "\n",
+    "- RDKit: a chemoinformatics package that wfl uses to convert 2D SMILES strings (e.g. \"CCO\" for ethanol) into 3D `Atoms` objects. \n",
+    "\n",
+    "  Documentation: https://rdkit.org/\n",
+    "\n",
+    "  Installation: `conda install -c conda-forge rdkit`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from ase.io import read, write\n",
+    "from ase import Atoms\n",
+    "\n",
+    "from xtb.ase.calculator import XTB\n",
+    "\n",
+    "from quippy.potential import Potential\n",
+    "\n",
+    "from wfl.configset import ConfigSet, OutputSpec\n",
+    "from wfl.generate import md\n",
+    "import wfl.descriptors.quippy\n",
+    "import wfl.select.by_descriptor\n",
+    "import wfl.fit.gap.simple\n",
+    "from wfl.calculators import generic\n",
+    "from wfl.autoparallelize.autoparainfo import AutoparaInfo\n",
+    "from wfl.autoparallelize.remoteinfo import RemoteInfo\n",
+    "from wfl.generate import smiles\n",
+    "from wfl.utils.configs import atomization_energy\n",
+    "from wfl.fit import error\n",
+    "from wfl.select.simple import by_bool_func\n",
+    "\n",
+    "from expyre.resources import Resources"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reference calculator \n",
+    "\n",
+    "To parallelise (non-file-based) calculators, they need to be given to workflow functions as \n",
```

Comment:
Technically this is only true for more complex ones, which can't be pickled (like quippy Potential).  Do we want to get into that level of detail here?  At least I'd change it to "the safest way to specify ... is to ..."